p2p/transport/websocket/listener.go

```diff
@@ -96,7 +100,7 @@ func (l *listener) serve() {
 
 func (l *listener) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	scheme := "ws"
-	if l.server.TLSConfig != nil {
+	if l.isWss {
```

Comment:
Should we fix the other checks as well?

Reply:
I guess there's just the other one in `serve`

Reply:
Done.